content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -163,7 +163,7 @@ times, or supply a comma-delimited list to `-S`:
 ```dvc
 $ dvc exp run -S learning_rate=0.001 -S units=128
 # is equivalent to
-$ dvc exp run -S learning_rate=0.001,units=128
+$ dvc exp run -S learning_rate=0.001 -S units=128
```

Comment:
I think we can just delete these lines since they would duplicate the line above, right?

Reply:
Made the change. PTAL.

Reply:
Lol yeah didn't read it haha. Gone